code/go/0chain.net/blobbercore/util/strings_test.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSplitPath(t *testing.T) {
+
+	tests := []struct {
+		name  string
+		path  string
+		items []string
+	}{
+		{
+			name:  "empty root",
+			path:  "/",
+			items: []string{},
+		},
+		{
+			name:  "only 1 directory level deep",
+			path:  "/file1",
+			items: []string{"file1"},
+		},
+		{
+			name:  "nested directories",
+			path:  "/dir1/dir2/dir3/file1",
+			items: []string{"dir1", "dir2", "dir3", "file1"},
+		},
```

Comment:
Adding cases for empty subpath perhaps, e.g. /dir1/dir2//dir3///dir4

Reply:
fixed